gettsim/social_insurance/krankenv.py

```diff
@@ -67,16 +65,16 @@ def ges_krankenv_beitr_m_tu(
     return ges_krankenv_beitr_m.groupby(tu_id).sum()
 
 
-def krankenv_beitr_regulär_beschäftigt(
-    lohn_krankenv_regulär_beschäftigt: FloatSeries, soz_vers_beitr_params: dict
+def krankenv_beitr_regulär_beschäft(
+    lohn_krankenv_regulär_beschäft: FloatSeries, soz_vers_beitr_params: dict
```

Comment:
For other social insurance stuff, isn't the same thing `bruttolohn_m_...` above? Should be consistent.

Reply:
good catch!